test/Lower/end-to-end-character-assignment.f90

```diff
@@ -1,3 +1,10 @@
+! RUN: bbc -emit-llvm -o - %s | tco | llc | sed 's/_QP//g' | as -o %t
+! RUN: c++ -std=c++17 %t %S/end-to-end-character-assignment-driver.cpp
```

Comment:
Nice way around the uniquer. lol

Reply:
Yes, that was the laziest way out, the one to mangle for compatibility with pgfortran/gfortran in expression tests is a bit uglier : `sed 's,_QP\([[:alnum:]_]*\),\1_,g'`